baseapp/baseapp.go

```diff
@@ -125,6 +125,10 @@ type BaseApp struct { // nolint: maligned
 	// indexEvents defines the set of events in the form {eventType}.{attributeKey},
 	// which informs Tendermint what to index. If empty, all events will be indexed.
 	indexEvents map[string]struct{}
+
+	// hooked services
+	// these hooks will have the ABCI messages routed through them
+	hooks []Hook
```

Comment:
"hooks" are just too broad of nomecnlature to use here IMO. Can we think of something more descriptive?